sql/timescaledb--0.6.1--0.7.0.sql

```diff
@@ -1,3 +1,8 @@
+CREATE OR REPLACE FUNCTION _timescaledb_internal.ddl_command_end() RETURNS event_trigger
```

Comment:
Same here, why not just `DROP EVENT TRIGGER`? Do we need it active during upgrades?

Now we are creating and replacing twice during the upgrade.